modules/manage/partials/tiered-storage.adoc

```diff
@@ -1250,6 +1250,50 @@ endif::[]
 
 == Disaster recovery
 
+With tiered storage enabled, you can restore your cluster, including its metadata, in the event of failure. This is a faster, less expensive alternative to active-active replication (for example, using xref:upgrade:migrate/data-migration.adoc[MirrorMaker 2]).
+
+You can take the following steps to restore a failed cluster:
+
+. Start a new cluster
+. Recover metadata from a failed source cluster to the new cluster
+. Recover topic data from the source cluster to the new cluster
+. Perform a rolling restart, if necessary
+. Direct application workload to the new cluster
+
+Use this recovery method to restore your application to the latest functional state as quickly as possible. Note that this method does not guarantee consistency at the application level. Restored transactions may not uphold atomicity. The consumer offset topic may not reflect point-in-time snapshots of topic data already consumed by applications. 
+
+=== Start a recovery target cluster
+
+Follow the steps to xref:deploy:deployment-option/cloud/index.adoc[provision a new cloud cluster]. Cloud account credentials (such as secrets or access keys) are not included in backed up metadata, and so you must manually configure these credentials for the new cluster.
+
+=== Cluster metadata recovery
+
+Metadata from a failed cluster can be restored to a target cluster using an rpk command. 
+
+[,bash]
+----
+rpk 
+----
+
+The following metadata is included in the restore:
+
+- Topic definitions. If you have enabled tiered storage only for specific topics, non-tiered storage will not be recovered 
+- Users and access control lists (ACLs)
+- Schemas
+- Consumer offsets 
```

Comment:
should we clarify 'including the  __consumer_offsets topic" @andrwng ? how best to say this?

Reply:
@mattschumpert @andrwng We also have [this doc](https://docs.redpanda.com/current/develop/consume-data/consumer-offsets/), let me know if I should just link to it here.

Reply:
>'including the __consumer_offsets topic" @andrwng ? how best to say this?

Yeah perhaps we can link that doc, and refer to it as the "consumer offsets topic"